client/src/components/RunCard/BigRunCard.js

```diff
@@ -75,21 +75,28 @@ const MapContainer = styled.div`
 const BigRunCard = ({
   isFavorite,
   onFavoriteClick,
-  date,
-  time,
   distance,
   runName,
+  startDate,
+  pace,
 }) => {
+  const parseDate = new Date(startDate).toLocaleString("de-DE", {
+    weekday: "short",
+    day: "2-digit",
+    month: "short",
+    hour: "2-digit",
+    minute: "2-digit",
+  });
```

Comment:
You can outsource this function if you like:

```suggestion
  const localeDateString = toLocaleDateString(startDate);
```

with
```js
const toLocaleDateString = (date, locale = 'de-DE') => {
  return new Date(date).toLocaleString(locale, {
    weekday: "short",
    day: "2-digit",
    month: "short",
    hour: "2-digit",
    minute: "2-digit",
  })
}
```

The other optimization is to use [`useMemo`](https://reactjs.org/docs/hooks-reference.html#usememo) to avoid calculation on every render.

```suggestion
  const localeDateString = useMemo(() => toLocaleDateString(startDate), [startDate]);
```